model/trace/gen-ai.yaml

```diff
@@ -56,32 +56,3 @@ groups:
           The `error.type` SHOULD match the error code returned by the Generative AI provider or the client library,
```

Comment:
> Next, right now there is no mention about tool errors, a type of message or anything. In Anthropic there is an is_error Do current implementors not care about this part, because for example everything is done sequentially and the error would bubble up into the calling span, or capturing it would be too expensive? If we don't care about errors, saying why raises less questions.

We should have a separate document that clarifies how generic conventions should apply to openai, anthropic, etc.

Check out how it's done for [databases](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/README.md) or messaging. 

There is an explicit clause allowing such extensions to add vendor-specific fields

> Semantic conventions for individual systems which extend content events SHOULD document all additional body fields and specify whether they
should be captured by default or need application to opt into capturing them.

Again, if you want to volunteer and send PR to add a document for anthropic, it'd be much appreciated.

This PR, again, does not change status quo - the prompts and completions were de-facto undefined for anthropic, now they are defined a bit more clearly. there is more work to do.